test/tests/smoke_testdata/idevid_csr.txt

```diff
@@ -16,6 +16,7 @@ Certificate Request:
                 ASN1 OID: secp384r1
                 NIST CURVE: P-384
         Attributes:
+<<<<<<< HEAD
```

Comment:
looks like merge markers are still in here

Reply:
Whoops. Updated.